src/Sylius/Bundle/WebBundle/Resources/views/Backend/Order/create.html.twig

```diff
@@ -11,15 +11,15 @@
 {% endblock %}
 
 {% block content %}
-<div class="page-header">
-    <h1><span class="glyphicon glyphicon-plus"></span> {{ 'sylius.ui.new_order'|trans|raw }}</h1>
-</div>
-
-{{ form_errors(form) }}
-
-<form action="{{ path('sylius_backend_order_create') }}" method="post" class="form-horizontal" novalidate>
-    <center>Not implemented yet.</center>
-    <hr>
-    {{ create() }}
-</form>
+{#<div class="page-header">#}
+    {#<h1><span class="glyphicon glyphicon-plus"></span> {{ 'sylius.ui.new_order'|trans|raw }}</h1>#}
+{#</div>#}
+{##}
+{#{{ form_errors(form) }}#}
+{##}
+{#<form action="{{ path('sylius_backend_order_create') }}" method="post" class="form-horizontal" novalidate>#}
+    {#<center>Not implemented yet.</center>#}
+    {#<hr>#}
+    {#{{ create() }}#}
+{#</form>#}
```

Comment:
I believe this template and corresponding routing and references can be removed.